GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -84,10 +84,11 @@ public final class StoryboardParser {
     storyboardsSegues[storyboardName] = delegate.segues
   }
 
-  public func parseDirectory(path: String) {
+  @available(OSX 10.10, *)
+  public func parseDirectory(path: String, osx: Bool = false) {
     if let dirEnum = NSFileManager.defaultManager().enumeratorAtPath(path) {
       while let subPath = dirEnum.nextObject() as? NSString {
-        if subPath.pathExtension == "storyboard" {
+        if subPath.pathExtension == "storyboard" && subPath.containsString("osx") == osx {
```

Comment:
This is still necessary but just for running tests. I didn't find a solution to get the right path from NSBundle. If you have a workaround to avoid this, I will definitely change it.

Otherwise, when used in a real case, the user will never have to add `osx` to their storyboard names, it will work transparently. It's really used only for the unit tests.